docs/toc.js

```diff
@@ -846,6 +846,11 @@ module.exports = {
           title: 'Reproduce',
           type: 'link',
         },
+        {
+          pathSegment: 'public-roadmap',
```

Comment:
Maybe just `roadmap` (with the content filename adjusted to match)?

I think the "public" is redundant with the fact that it's published on our docs.